tests/e2e/specs/price-benchmark/price-benchmark-suggestions.test.js

```diff
@@ -27,18 +26,59 @@ test.describe( 'Price Benchmark Page', () => {
 	test.beforeAll( async ( { browser } ) => {
 		page = await browser.newPage();
 		priceBenchmarkPage = new PriceBenchmarkPage( page );
-		await Promise.all( [ priceBenchmarkPage.mockRequests() ] );
 	} );
 
 	test.afterAll( async () => {
-		await clearOnboardedMerchant();
 		await page.close();
 	} );
 
+	test.describe( 'Price Benchmark Tour', () => {
+		test.beforeAll( () => {
+			priceBenchmarkPage.fulfillGetTour( [
+				{
+					id: 'price-benchmark-tour',
+					checked: true,
```

Comment:
@ankitrox Should this not be `false`?  We can even add a new function similar to `mockCompletedTour` where we set checked to `false`.